indexdoc.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+ <head>
+   <meta charset="UTF-8">
+   <link rel="stylesheet" href="styles.css">
+   <title>Technical Documentation Page</title>
+ </head>
+
+ <body>
+   <main>  
+ <main id="main-doc" >JS Documentation
+<section id="Introduction" class="main-section">
+  <header>Introduction</header>
+  <p>JavaScript is a cross-platform, object-oriented scripting language. It is a small and lightweight language. Inside a host environment (for example, a web browser), JavaScript can be connected to the objects of its environment to provide programmatic control over them.</p>
+  <p>JavaScript contains a standard library of objects, such as Array, Date, and Math, and a core set of language elements such as operators, control structures, and statements. Core JavaScript can be extended for a variety of purposes by supplementing it with additional objects; for example:</p>
+  <ul>
+    <li><p>Client-side JavaScript extends the core language by supplying objects to control a browser and its Document Object Model (DOM). For example, client-side extensions allow an application to place elements on an HTML form and respond to user events such as mouse clicks, form input, and page navigation.</p></li>
+    <li><p>Server-side JavaScript extends the core language by supplying objects relevant to running JavaScript on a server. For example, server-side extensions allow an application to communicate with a database, provide continuity of information from one invocation to another of the application, or perform file manipulations on a server.</p></li>
+  </ul>
+</section>
+<section id="What_you_should_already_know" class="main-section">
+  <header>What you should already know</header>
+  <p>This guide assumes you have the following basic background:
+</p>
+<ul>
+  <li><p>A general understanding of the Internet and the World Wide Web (WWW).</p></li>
+  <li><p>Good working knowledge of HyperText Markup Language (HTML).</p></li>
+  <li><p>Some programming experience. If you are new to programming, try one of the tutorials linked on the main page about JavaScript.</p></li>
+</ul>
+</section>
+<section id="JavaScript_and_Java" class="main-section">
+  <header>JavaScript and Java</header>
```

Comment:
- You have phrased the comments politely. :+1: 
- However, to perform code review, we need to constantly explain "why" we request a change - it might be evident to us but not to the code author, please let's provide a "why", for this requested change as already mentioned by the previous reviewer.

e.g saying :point_down: 

**It is a better practice to use hx (h1, h2, h3 . . . h6 ) for the tiles of every section instead of a header, that is semantically more appropriate and will also increase the site's accessibility you can also provide a [link to html semantics](https://developer.mozilla.org/en-US/docs/Glossary/Semantics)**